docs/appendices/release-notes/unreleased.rst

```diff
@@ -70,6 +70,12 @@ Fixes
 .. stable branch. You can add a version label (`v/X.Y`) to the pull request for
 .. an automated mergify backport.
 
+- Changed the logic to resolve functions. Previously it would first look for
```

Comment:
What do you think about adding a `BREAKING CHANGE` pointing here, just in case it affects some users?

Reply:
If we classify it as a breaking change it can't go into a hotfix. It's either a fix or a breaking change, but not both. 

I have no strong opinion in either direction, but slight tendency towards fix

Reply:
yep, I get it, let's leave it as fix, thx!